server/controllers/refer/peopleController.test.ts

```diff
@@ -0,0 +1,68 @@
+import type { DeepMocked } from '@golevelup/ts-jest'
+import { createMock } from '@golevelup/ts-jest'
+import type { NextFunction, Request, Response } from 'express'
+import createError from 'http-errors'
+
+import PeopleController from './peopleController'
+import type { Person } from '../../@types/models/Person'
+import type PersonService from '../../services/personService'
+import { personFactory } from '../../testutils/factories'
+import personUtils from '../../utils/personUtils'
+
+describe('PeopleController', () => {
+  const token = 'SOME_TOKEN'
+  const request: DeepMocked<Request> = createMock<Request>({ user: { token } })
+  const response: DeepMocked<Response> = createMock<Response>({})
+  const next: DeepMocked<NextFunction> = createMock<NextFunction>({})
+
+  const personService = createMock<PersonService>({})
+
+  let peopleController: PeopleController
+
+  beforeEach(() => {
+    peopleController = new PeopleController(personService)
+  })
+
+  describe('show', () => {
+    let person: Person
+
+    beforeEach(() => {
+      person = personFactory.build({
+        name: 'Del Hatton',
+      })
```

Comment:
Does this need to happen before each rather than than once as a `const` in the `describe` block?